gdal/ogr/ogrsf_frmts/flatgeobuf/ogr_flatgeobuf.h

```diff
@@ -86,7 +86,7 @@ class OGRFlatGeobufLayer final : public OGRLayer
         uint64_t m_offset = 0;
         uint64_t m_offsetFeatures = 0;
         uint64_t m_offsetIndices = 0;
-        std::vector<uint64_t> m_foundFeatureIndices;
+        std::vector<std::pair<uint64_t, uint64_t>> m_foundFeatureIndexOffsets;
```

Comment:
a few comments about the meaning of those variables wouldn't hurt. As far as I understand m_featuresPos goes from 0 to m_featuresSize - 1, and m_featuresSize is the number of features that match the spatial index ? and the (usable) size of m_foundFeatureIndexOffsets is m_featuresSize ?

Reply:
it might perhaps help to create a struct IndexOffset { size_t index; uint64_t fileOffset } and use that instead of a std::pair

Reply:
shouldn't m_featuresPos and m_featuresSize be of size_t, since they represent a number of things that must hold in RAM right ? That could probably save a few static_cast<size_t>

Reply:
Agreed, I will try to make some meaningful comments and rework the pair. However, m_featuresSize is actually some left over that is not used - I assume you mean m_featuresCount?

Reply:
> I assume you mean m_featuresCount?

indeed !